HULKs/hulk
# Drop obstacles on Unpenalized

## Introduced Changes

drops all obstacles in the filter, when leaving a penalty.

Based on #190 #191 #198 

## ToDo / Known Issues

none

## Ideas for Next Iterations (Not This PR)

none

## How to Test

Use your hand to generate a sonar obstacle, press the chest button to leave penalized state, the obstacle should be removed
